_includes/_components/url-generation.html

```diff
@@ -35,16 +50,37 @@
 {% highlight php %}
 <?php
 
-class Test
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+
+class TestController
 {
-    public function test(SomeClass $subject)
+    /**
+     * @Route("/s/{shopId}", name="shop_show")
+     */
+    public function showAction($shopId)
     {
-        return $subject->doSomething();
+        return $this->redirectToRoute('shop_products');
+    }
+
+    /**
+     * @Route("/s/{shopId}/p", name="shop_products")
+     */
+    public function productsAction($shopId)
+    {
+        // ...
     }
 }
 {% endhighlight %}
               </div>
             </div>
           </div>
+          <p>
+              Works also with Twig `url` and `path` functions or everiwhere else you use the Symfony router.
```

Comment:
Works also with `url` and `path` Twig functions or everywhere else you use the Symfony router.